imports/ui/unit/unit.jsx

```diff
@@ -83,12 +143,84 @@ class Unit extends Component {
       })
     }
   }
+  handleReportFilterClicked = (event, index, statusFilterValues) => {
+    this.setState({
+      statusFilterValues: statusFilterValues
+    })
+  }
+
+  handleReportSortClicked = (event, index, value) => {
+    this.setState({
+      sortBy: value
+    })
+  }
+
+  reportFilterMenu (statusFilterValues) {
+    const status = ['Draft', 'Finalized', 'Created By Me']
+    return status.map((name) => (
+      <MenuItem
+        key={name}
+        insetChildren
+        checked={statusFilterValues && statusFilterValues === {name}}
+        value={name}
+        primaryText={name}
+      />
+    ))
+  }
+
+  reportSortMenu (sortBy) {
+    const labels = new Map([
```

Comment:
The `Map` constructor seems a bit redundant here, as it's being immediately converted to array, and not used for anything else. It would be better to keep it as simple as possible by just creating it as an array from the start